recipe-server/client/control_new/routes.js

```diff
@@ -22,16 +24,16 @@ const routes = {
         crumb: 'View Recipe',
       },
     },
-    '/extensions': {
+    '/extension': {
```

Comment:
Two things:
- This needs a trailing slash to stay consistent with other routes.
- Links in the `Gateway` component point to `/extensions`, which should be fixed. (And in the nav menu, but that's in another open PR, so I'll fix that.)